recipes/pangolin/meta.yaml

```diff
@@ -29,10 +29,11 @@ requirements:
     - snakemake-minimal >=5.13
     - gofasta
     - usher >=0.3.5
-    - pangolearn >=2021.08.09
+    - pangolearn >=2021-08-24
```

Comment:
Shouldn't this be `==`?

Reply:
The upstream package has >= so I think it should be kept this way.